examples/tutorial/02-ETL-with-NVTabular.ipynb

```diff
@@ -27,6 +27,21 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    # from nvtabular import ColumnSelector
</pre><p>We could remove this commented line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/467/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>